lib/libsys/Symbol.map

```diff
@@ -1043,11 +1043,13 @@ FBSDprivate_1.0 {
 	__set_error_selector;
 	nlm_syscall;
 	gssd_syscall;
-	__libc_interposing_slot;
-	__libc_sigwait;
 	_cpuset_getdomain;
 	__sys_cpuset_getdomain;
 	_cpuset_setdomain;
 	__sys_cpuset_setdomain;
 	rpctls_syscall;
+
+	__libc_interposing_slot;
+	__libc_sigwait;
+	__set_error_selector;
```

Comment:
why the reordering?

Reply:
In later commits all the `_foo` and `__sys_foo` syscall symbols move to a generate file. There should probably be a preparatory commit that splits the block into "standard syscall stuff" and "everything else".